[INFO -- 2017-02-26 19:11:39]
Starting flip_value script
version: 2.0
[INFO -- 2017-02-26 19:11:39]
Initialization strings:
[INFO -- 2017-02-26 19:11:39]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-02-26 19:11:39]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 19:11:39]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 19:11:39]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 19:11:39]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 19:11:39]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-02-26 19:11:39]
initSignal:4
[INFO -- 2017-02-26 19:11:39]
endSignal:8
[INFO -- 2017-02-26 19:11:39]
seqSignal:30
[INFO -- 2017-02-26 19:11:39]
faultMode:2
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 19:11:49]
pythonException: invalid frame
[INFO -- 2017-02-26 19:11:49]
Backtrace BEGIN:
#0  0x0000000000404caf in L_lud_omp_123__par_loop2_2_5 () at lud_omp.c:148
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffd9c0 in ?? ()
#3  0x00007ffff4ea0100 in ?? ()
#4  0x0000000000000001 in ?? ()
#5  0x0000000000000003 in ?? ()
#6  0x0000000000000000 in ?? ()

143	            for (i = 0; i < BS; i++)
144	            {
145	                for (k=0; k < BS; k++) {
146	#pragma omp simd 
147	                    for (j = 0; j < BS; j++) {
148	                        sum[j] += temp_left[BS*i + k] * temp_top[BS*k + j];
149	                    }
150	                }
151	#pragma omp simd 
152	                for (j = 0; j < BS; j++) {

Backtrace END
Memory content before bitflip:0x500x030x000x00
Fault Mode: Random bit-flip
base address to flip value: 0x7fffffffd8f8
address max offset: 4
Memory content after  bitflip:0xa70x6a0x1c0xb3
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: offset
symbol filename: lud_omp.c
symbol line: 45
value: -1289983321
value address: 0x7fffffffd8f8
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-02-26 19:11:49]
Fault Injection Successful after 10.6642129421s
[INFO -- 2017-02-26 19:11:51]
event type: exit
[EXCEPTION -- 2017-02-26 19:11:51]
exit code: no exit code available
